addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -8394,10 +8394,17 @@ registry.many2one = SnippetOptionWidget.extend({
  * Allows to display a warning message on outdated snippets.
  */
 registry.VersionControl = SnippetOptionWidget.extend({
+    events: {
+        "click .o_snippet_replace": "_onReplaceSnippetClick",
+        "click .o_access_options": "_onAccessOptionsClick",
+    },
     /**
      * @override
      */
     start: function () {
+        if (this.$target[0].isAlertDiscarded) {
```

Comment:
Do you need to save that information on the `$target` element ? Can't this be `this.isAlertDiscarded`?

Reply:
This is what I wanted to do at first (because I know you don't like storing things in the DOM haha) but it was not possible here because as said in the comment about the start, it restarts every time and so the `this` was lost. So I needed to store this info in something that will not change :slightly_smiling_face: Note that it is just there during edit mode: from what I saw, it is removed when we save